content/maxwell1_fundamentals/maxwells_equations/maxwells_equations_in_homogeneous_media/dipole_sources/electric_dipole_definition/index.rst

```diff
@@ -0,0 +1,122 @@
+.. _definition_electric_dipole_index:
+
```

Comment:
can you please remove these files from this pull request? They should be included in a seperate pr.